### PR TITLE
feature/youtube-channel-template

### PR DESCRIPTION
**Pull request by FranDRoss**

### CODE REVIEW

**Comment by paulusrex** [comment]
```diff
diff --git a/staff/francesc-destruels/youtube-channel-template/index.html b/staff/francesc-destruels/youtube-channel-template/index.html
index 38e31b7..6e9394c 100644
--- a/staff/francesc-destruels/youtube-channel-template/index.html
+++ b/staff/francesc-destruels/youtube-channel-template/index.html
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="./style.css" />
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
         integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
     <title>SillyBilly</title>
 </head>
 
diff --git a/staff/francesc-destruels/youtube-channel-template/style.css b/staff/francesc-destruels/youtube-channel-template/style.css
index b45dd42..e4a2640 100644
--- a/staff/francesc-destruels/youtube-channel-template/style.css
+++ b/staff/francesc-destruels/youtube-channel-template/style.css
@@ -1,10 +1,12 @@
 body {
     margin: 0;
     height: 100vh;
+    font-family: 'Roboto', sans-serif;
 }
 
 * {
     box-sizing: border-box;
+
 }
 
 a {

```

**Comment by paulusrex** [comment]
You can put the original font for youtube with the patch above

File: staff/francesc-destruels/youtube-channel-template/index.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+                <p class="t6">
+                    Hola<br /> Este video esta muy<br /> chulo!
+                </p>
+                <i class="fas fa-ellipsis-v i6"></i>
+            </section>
+
+        </div>
+        <div class="links linksGrind">
+
+                <img id="img" class="linksPics" alt="icon"
```
Comment (mlila) [inline staff/francesc-destruels/youtube-channel-template/index.html]: @FranDRoss you can't use the same id in different components

Comment (FranDRoss) [reply staff/francesc-destruels/youtube-channel-template/index.html]: I am not using it at all, I forgot to erase it

**Pull request merged**